cfgov/jinja2/v1/external-site/index.html

```diff
@@ -31,6 +31,12 @@
     {# No footer needed on this page. #}
 {%- endblock %}
 
+{% block javascript %}
+<script type='text/javascript'>
+    {% include '/js/externalSite.js' %}
```

Comment:
TRIVIAL: should probably be `external-site.js` to follow naming style of other non-class modules in the project.